src/net/libwebsocketsIO.cpp

```diff
@@ -282,6 +282,7 @@ void LibwebsocketsClient::resetOutputBuffer()
 #define EVP_PKEY_get0_RSA(_pkey_) ((_pkey_)->pkey.rsa)
 #endif
 
+#ifdef USE_WEBRTC
```

Comment:
I think these methods actually belongs to OpenSSL. When building with WebRTC, the crypto library is replaced by BoringSSL, which is already part of WebRTC library, and it's used for the SDK as well, as replacement of OpenSSL. However, without WebRTC, I think those methods should be kept.
What is exactly the issues with them on Windows?

Reply:
Actually, this change breaks building on Linux. 
![image](https://user-images.githubusercontent.com/11694066/90485344-b25b4c00-e137-11ea-89b7-723387a52be9.png)
The method below, ` static bool check_public_key(X509_STORE_CTX* ctx)` needs them.

Reply:
On windows the symbol is doubly defined - one of the 3rdParty libraries already has them.   I'll change it to #ifndef WIN32 then, thanks